content/release-notes/2022-09-02.md

```diff
@@ -4,10 +4,10 @@ label: 'Console'
 
 ### What's new
 
-- UI: Added support for passing the OAuth app name to the client.
+- UI: Added support for passing an OAuth app name to a client.
 
 ### Bug fixes
 
 - UI: Fixed the title on the 'Sign in' page.
 - UI: Changed the PostgreSQL version displayed on the project dashboard to 14.5.
-- Control Plane: Fixed authentication of concurrent proxy connections to the idle compute node.
+- Control Plane: Fixed authentication of concurrent proxy connections to an idle compute node.
```

Comment:
What was the symptom for the users? Maybe we should outline that the symptom was fixed to make it eaier to relate to this?

Reply:
Added more context: "Fixed authentication of concurrent proxy connections to an idle compute node. Previously, one of the concurrent proxy connection attempts would fail with a `Failed to connect to the compute node` error."